example/basic/main.go

```diff
@@ -22,6 +22,13 @@ import (
 	"github.com/getsentry/sentry-go"
 )
 
+// release is the release of this program that will be reported to Sentry.
+// Use go build -ldflags='-X main.release=VALUE' to set this value in build
+// time.
+// If not set, a default release value will be derived in runtime from
+// environment variables or the Git repository in the current working directory.
+var release string
```

Comment:
If we're happy with this suggestion, I'd add the same to other examples, and a similar bit of explanation to https://docs.sentry.io.